p4src/v1model/include/control/next.p4

```diff
@@ -279,6 +279,7 @@ control EgressNextControl (inout ingress_headers_t        hdr,
     apply {
         if (fabric_md.bridged.base.is_multicast
              && fabric_md.bridged.base.ig_port == standard_md.egress_port) {
+            fabric_md.bridged.int_bmd.report_type = INT_REPORT_TYPE_NO_REPORT;
```

Comment:
Wait... I thought we agreed on generating a DROP report with specific reason. I remember @ccascone suggested looking into existing reason.

Reply:
Carmelo and I discussed this on [slack](https://onf-internal.slack.com/archives/C022483HJ78/p1646341515675969), we think we can ignore reporting this packet drop.